Sources/TensorFlow/Core/Runtime.swift

```diff
@@ -35,6 +35,7 @@ import Darwin
 import Glibc
 #endif
 import CTensorFlow
+import Foundation
```

Comment:
Is `Foundation` or Grand Central Dispatch required? Note that the the build on the apple/swift side isn't configured to depend on Foundation or Dispatch, which is also why we used pthread instead of `Dispatch` in the compiler runtime.

Reply:
Interesting. It's only used in this block:
```swift
signal(SIGINT, SIG_IGN)
let sigintSrc = DispatchSource.makeSignalSource(signal: SIGINT, queue: .main)
sigintSrc.setEventHandler {
    print("Caught interrupt signal, exiting...")
    exit(1)
}
sigintSrc.resume()
```
I switched to using `import Dispatch` in the last commit. Is that fine?

Reply:
I don't think apple/swift can depend on `Dispatch` either, unfortunately. Is there a way to use neither of them? 

A long term solution for our toolchain build is to follow how swift-corelibs-foundation is getting included in Swift toolchain builds, in which case we should be able to rely on `Dispatch` and `Foundation`.

Reply:
This is only really necessary for the `SIGINT` handler. Is there a way to provide a closure for the handler directly in `signal(SIGINT, ...)` (in place of `...`)?

Reply:
You can use Swift's closure syntax directly. A closure can be implicitly converted to a C function pointer (aka. a `@convention(c)` function) if the closure 1) does not capture anything and 2) uses C-compatible types in the signature.

Reply:
You can try 
```swift
signal(SIGINT) { _ in
    print("Caught interrupt signal, exiting...")
    exit(1)
}
```

Reply:
Thanks, that's cool and it works :) Just pushed it.